server/src/main/java/org/apache/druid/server/metrics/QueryCountStatsMonitor.java

```diff
@@ -44,11 +44,16 @@ public QueryCountStatsMonitor(
   public boolean doMonitor(ServiceEmitter emitter)
   {
     final ServiceMetricEvent.Builder builder = new ServiceMetricEvent.Builder();
+    final long successfulQueryCount = statsProvider.getSuccessfulQueryCount();
+    final long failedQueryCount = statsProvider.getFailedQueryCount();
+    final long interruptedQueryCount = statsProvider.getInterruptedQueryCount();
     Map<String, Long> diff = keyedDiff.to(
         "queryCountStats",
-        ImmutableMap.of("query/success/count", statsProvider.getSuccessfulQueryCount(),
-                        "query/failed/count", statsProvider.getFailedQueryCount(),
-                        "query/interrupted/count", statsProvider.getInterruptedQueryCount()
+        ImmutableMap.of("query/count", successfulQueryCount + failedQueryCount + interruptedQueryCount,
```

Comment:
Should break `"query/count", successfulQueryCount + failedQueryCount + interruptedQueryCount,` on the next line, as well as the following lines

Reply:
Oops, changed.